src/components/prescriptions/PrescriptionComponent.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <li
+        v-bind:key="props.id"
+        class="documentBox">
+        <font-awesome-icon :icon="classOfIcon" />
+        <h5 class="documentBoxtitle">{{ name }}</h5>
```

Comment:
redundant element. Semanticlly wrong beacuse Name and value has both h5 markup and the same class